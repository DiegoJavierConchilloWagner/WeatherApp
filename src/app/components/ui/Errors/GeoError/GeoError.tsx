/* eslint-disable react/no-unescaped-entities */
import { FC } from 'react';

import { WeatherIcon } from '@components/WeatherIcon/WeatherIcon';
import { StyledMessage } from '../styled';

export const GeoError: FC = () => {
	return (
		<StyledMessage>
			<WeatherIcon id={200} icon='n' size={200} isLoading={false} />
			<h1>
				There was a problem getting your curren position but don't worry, you can search for a place to see its
				weather.
			</h1>
		</StyledMessage>
	);
};
