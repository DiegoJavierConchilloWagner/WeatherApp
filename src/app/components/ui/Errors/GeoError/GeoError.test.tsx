import renderWithThemeAndProvider from '@common/utils/renderWhitThemeAndProvider';
import { GeoError } from './GeoError';

describe('Example GeoError', () => {
	it('should render correcty', () => {
		const wrapper = renderWithThemeAndProvider(<GeoError />);

		expect(wrapper).toMatchSnapshot();
	});
});
