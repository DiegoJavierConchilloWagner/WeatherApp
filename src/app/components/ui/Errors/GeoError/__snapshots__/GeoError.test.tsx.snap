// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example GeoError should render correcty 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  margin: 2rem 5rem 2rem 5rem;
  gap: 2rem;
}

.c0 img {
  height: 200px;
}

.c0 h1 {
  font-size: 1.2rem;
  font-weight: 400;
}

<body>
    <div>
      <div
        class="c0"
      >
        <img
          alt="weather"
          height="200"
          src="SvgrURL"
          width="200"
        />
        <h1>
          There was a problem getting your curren position but don't worry, you can search for a place to see its weather.
        </h1>
      </div>
    </div>
  </body>,
  "container": .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  margin: 2rem 5rem 2rem 5rem;
  gap: 2rem;
}

.c0 img {
  height: 200px;
}

.c0 h1 {
  font-size: 1.2rem;
  font-weight: 400;
}

<div>
    <div
      class="c0"
    >
      <img
        alt="weather"
        height="200"
        src="SvgrURL"
        width="200"
      />
      <h1>
        There was a problem getting your curren position but don't worry, you can search for a place to see its weather.
      </h1>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
